src/gui/widgets/MixerChannelLcdSpinBox.cpp

```diff
@@ -40,6 +43,13 @@ void MixerChannelLcdSpinBox::setTrackView(TrackView * tv)
 
 void MixerChannelLcdSpinBox::mouseDoubleClickEvent(QMouseEvent* event)
 {
+	if (!(event->modifiers() & Qt::ShiftModifier) &&
+		!(event->modifiers() & Qt::ControlModifier))
+	{
+		enterValue();
+		return;
+	}
+
```

Comment:
Could you explain to me why exactly we are doing this? It isn't wrong necessarily, but I want to understand a bit.

Reply:
To be specific, I was wondering why double clicking with Shift and Control where blocked.

Reply:
If double clicking with Shift and Control, it will show the input dialog instead of opening the mixer.